user/advanced-topics/bind-dirs.md

```diff
@@ -24,34 +24,42 @@ In this way sys-whonix can benefit from the Tor anonymity feature 'persistent To
 
 ## How to use bind-dirs.sh? ##
 
-In this example, we want to make `/var/lib/tor` persistent.
+In this example, we want to make `/var/lib/tor` persistent. Enter all of the following commands in your app qube.
 
-Inside the app qube.
-
-1. Make sure folder `/rw/config/qubes-bind-dirs.d` exists.
+1. Make sure the directory `/rw/config/qubes-bind-dirs.d` exists.
 
    ```
    sudo mkdir -p /rw/config/qubes-bind-dirs.d
    ```
 
-2. Create a file `/rw/config/qubes-bind-dirs.d/50_user.conf` with root rights.
+2. Create the file `/rw/config/qubes-bind-dirs.d/50_user.conf` with root permissions, if it doesn't already exist.
+
+   ```
+   sudo touch /rw/config/qubes-bind-dirs.d/50_user.conf
+   ```
 
-3. Edit the file 50_user.conf to append a folder or file name to the `binds` variable.
+3. Add a line to `/rw/config/qubes-bind-dirs.d/50_user.conf` that appends a folder or file to the `binds` variable.
 
    ```
    binds+=( '/var/lib/tor' )
    ```
 
 4. Save.
 
-5. Reboot the app qube.
+5. If the directory you wish to make persistent doesn't exist in the template on which the app qube is based, you'll need to create the directory (with its full path) under `/rw/bind-dirs` in the app qube. For example, if `/var/lib/tor` didn't exist in the template, then you would execute the following command in your app qube:
+
+   ```
+   sudo mkdir -p /rw/bind-dirs/var/lib/tor
```

Comment:
Maybe it should be mentioned that the permissions of these directories will be reused by bind-dirs, so you may have to take care of owner/group/permissions.

Reply:
That makes sense. But I'm not sure what expectations bind-dirs has regarding owner/group/permissions.

Reply:
The script will simply mount `/rw/bind-dirs/foobar` on top of `/foobar`, inheriting the permissions of `/rw/bind-dirs/foobar`.

So, if `/foobar` is owned by root:user and you mount `/rw/bind-dirs/foobar` owned by user:user, the latter is what you will see on `/foobar`

Reply:
good :) thanks

just to be pedantic, this can be done in one shot using `install -d -D -o owner -g group -m 755 /rw/bind-dirs/var/lib/tor`

Reply:
Good to know! Although, I think I'll keep it as-is to make the commands simpler.

Reply:
I also think it should be kept as understandable as possible, user with basics skills should know about chown / chmod